BedrockPlugin.cpp

```diff
@@ -2,7 +2,7 @@
 #include "BedrockPlugin.h"
 
 // Global static values
-list<BedrockPlugin*>* BedrockPlugin::g_registeredPluginList = 0;
+list<BedrockPlugin*>* BedrockPlugin::g_registeredPluginList = nullptr;
```

Comment:
I actually like `nullptr`. It's a new C++11 feature that actually is semantically significant, as opposed to the old `NULL` macro, that I agree didn't add much.

Here's a description of `nullptr`: http://www.devx.com/cplus/10MinuteSolution/35167/0/page/2

I do like making methods `const` when they can be, because if nothing else it lets you call them from other `const` methods, but agree with @quinthar that in this case it's not applicable because of the mutex that needs to be locked.

I don't have strong opinions on `size_t`.